docs/locales/hi/LC_MESSAGES/index.po

```diff
@@ -0,0 +1,27 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) <a href="https://d.pali.tools/">Digital Pāli Tools</a>
+# This file is distributed under the same license as the Digital Pāli Reader
+# package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2021.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: Digital Pāli Reader \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-09-10 10:25-0400\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.9.1\n"
+
+#: ../../index.rst:4 2bd53e8a8a224279949877f0435453fa
+msgid "Getting Started"
+msgstr "मदद"
```

Comment:
Here I am adding the placeholder text "Help" (translation provided by Google Translate) to show an example of how to maintain this PO file; `msgid` contains the source text (English) and `msgstr` contains the translated text. (These PO files are auto-generated from the source documentation; see https://docs.readthedocs.io/en/stable/guides/manage-translations.html.)

I have left the remaining `msgstr` elements empty for now, and they can be filled in by future translators. (Note that the default behavior of an empty `msgstr` is to show the original source text instead, _i.e._ English.)